bot/extensions/polls/views.py

```diff
@@ -0,0 +1,102 @@
+import discord
+from discord import ui
+
+
+class PollModal(ui.Modal):
+    name = ui.TextInput(label="Choice name", placeholder="Enter poll choice", max_length=50, required=True)
+
+    def __init__(self, var: discord.Interaction):
+        self.var = var
+        super().__init__(title="Poll options")
```

Comment:
Modals and Views should preferably be stateless
This means `timeout=None` and having custom_id on actions

Should also be able to pass title like we've said to do in discord.